### PR TITLE
test: Created unit tests to implement native ethers transfer 

### LINKED ISSUE #206: test: create unitary tests to implement native ether transfers in `TestSwaplace.test.ts`

### PR DESCRIPTION
**Pull request by blackbeard002**
closes #206

### CODE REVIEW

File: test/TestSwaplace.test.ts [hunk 1]
```diff
@@ -466,6 +538,69 @@ describe("Swaplace", async function () {
...
+
+        const askingAddr = [
+          MockERC20.address,
+          MockERC20.address,
+          MockERC20.address,
+        ];
+        const askingAmountOrId = [50, 100, 150];
+
+        const valueToSend: BigNumber = ethers.utils.parseEther("0.5");
+
```
Comment (0xneves) [inline test/TestSwaplace.test.ts:560]: Whitespace can be removed

File: test/TestSwaplace.test.ts [hunk 2]
```diff
@@ -553,6 +688,60 @@ describe("Swaplace", async function () {
           ),
         ).to.be.revertedWithCustomError(Swaplace, "InvalidAddress");
       });
+
+      it("Should revert when wrong amount of ethers are sent by the {acceptee}", async function () {
```
Comment (0xneves) [inline test/TestSwaplace.test.ts:692]: It lacks a test where the acceptee successfully accepts sending the ether

**Review by 0xneves — CHANGES_REQUESTED** [review]
Requesting minor changes in the name of the tests and whitespace removal. Also requiring a missing test

File: test/TestSwaplace.test.ts (hunk 2 above)
Comment (blackbeard002) [reply test/TestSwaplace.test.ts:692]: > It lacks a test where the acceptee successfully accepts sending the ether

It's quite ambiguous since the acceptee can either accept or send ether. Did you mean to refer to your comment `Should verify with expect if the value was transferred as expected` or did you mean to say an entire new test where the `acceptee` successfully sends ethers to the `owner`.

Comment (0xneves) [reply test/TestSwaplace.test.ts:692]: I had no place to input my request since the test was non-existent. There are no tests in which the one sending the ether as `msg.value` is the `acceptee` and the swap is accepted. There is thought, a test where the accepted sends ether and it fails, which is the test I pointed out above. The test lacking is the successful version of this failed one.

Comment (blackbeard002) [reply test/TestSwaplace.test.ts:692]: Perfect! Onto it

**Comment by blackbeard002** [comment]
Hey @0xneves made the requested changes. Ready for review

File: test/TestSwaplace.test.ts [hunk 3]
```diff
@@ -339,7 +339,7 @@ describe("Swaplace", async function () {
         const askingAmountOrId = [50, 100, 150];
 
         const valueToSend: BigNumber = ethers.utils.parseEther("0.5");
-
+
```
Comment (0xneves) [inline test/TestSwaplace.test.ts:342]: Unnecessary whitespace was added here

**Review by 0xneves — CHANGES_REQUESTED** [review]
Requesting minor change:

File: test/TestSwaplace.test.ts (hunk 3 above)
Comment (blackbeard002) [reply test/TestSwaplace.test.ts]: @0xneves sorry, removed it now

**Review by 0xneves — APPROVED** [review]
Approving these changes
